packages/one-app-bundler/bin/postProcessOneAppBundle.js

```diff
@@ -26,12 +28,22 @@ const tmpPath = path.resolve(pkgPath, '../build/app/tmp');
 
 module.exports = async function postProcessBuild() {
   const endsWithJS = (fileName) => fileName.endsWith('.js');
-  const legacyPath = path.join(tmpPath, 'legacy');
+  const configOptions = getConfigOptions();
+  const disableDevelopmentLegacyBundle = configOptions.disableDevelopmentLegacyBundle && process.env.NODE_ENV === 'development';
```

Comment:
```suggestion
  const disableDevelopmentLegacyBundle = getConfigOptions().disableDevelopmentLegacyBundle && process.env.NODE_ENV === 'development';
```
Since its not being used below do we need to create a var ?